xclim/sdba/base.py

```diff
@@ -29,12 +29,21 @@ def __init__(self, **kwargs):
             setattr(self, key, val)
             self._parameter_names.append(key)
 
+    def _add_parameter(self, name, value):
+        setattr(self, name, value)
+        self._parameter_names.append(name)
+
```

Comment:
Why not just subclass dict ? I have the feeling this is over-engineered.

Reply:
That could be done... The idea was to allow subclasses to declare attributes that wouldn't be considered as "parameters". But I got into this problems with DQM where the sub-subclass adds a parameter to the subclass (and so `super().__init__()` is not enough...).